content/docs/pipelines/sdk/enviroment_variables.md

```diff
@@ -0,0 +1,73 @@
++++
+title = "Using enviroment varibales in piplines"
+description = "How set and use enviroment variables in kubeflow piplines"
++++
+
+This page describes how to create a enviroment varaibale for Kubeflow Pipelines 
+and how to combine enviroment variable into a pipeline. 
```

Comment:
Maybe you could change the last part. It sounds like the env variables create a pipeline.

Reply:
Will change, see what you mean!